packages/internal/src/types.ts

```diff
@@ -58,15 +58,14 @@ export declare class SyncCallback<
 
 /**
 Returns a boolean for whether the given type is primitive value or primitive type.
- 
 @example
 ```
 IsPrimitive<'string'>
 //=> true
- 
+
 IsPrimitive<string>
 //=> true
- 
+
 IsPrimitive<Object>
 //=> false
 ```
```

Comment:
```suggestion
/**
 * Returns a boolean for whether the given type is a primitive value or
 * primitive type.
 *
 * @example
 * ```
 * IsPrimitive<'string'>
 * //=> true
 *  
 * IsPrimitive<string>
 * //=> true
 *  
 * IsPrimitive<Object>
 * //=> false
```

Reply:
Done as a drive-by. Doesn't actually have anything to do with this PR. Thanks.